libraries/standard/mqtt/include/mqtt.h

```diff
@@ -160,9 +160,14 @@ MQTTStatus_t MQTT_Init( MQTTContext_t * pContext,
  * @brief param[in] pContext Initialized MQTT context.
  * @brief param[in] pConnectInfo MQTT CONNECT packet parameters.
  * @brief param[in] pWillInfo Last Will and Testament. Pass NULL if not used.
- * @brief param[in] timeoutMs Timeout in milliseconds for receiving
- * CONNACK packet. A zero timeout value will attempt a single reception
- * from the transport interface.
+ * @brief param[in] timeoutMs Timeout in milliseconds for receiving CONNACK
+ * packet. In certain conditions, depending on the network conditions or the
+ * transport interface receive timeout configured, this function can spend more
+ * time in receiving CONNACK packet as compared to the timeout provided. However
+ * the probability of exceeding the timeout is very low assuming the the timeouts
+ * for CONNACK receive and transport interface receive are configured correctly.
+ * Refer to the transport interface documentation for more details about the
+ * timeouts. A zero timeout will make use of the retries for receiving CONNACK.
```

Comment:
Maybe a `@note` tag for the last sentence can be easier to notice about use of retries for the zero timeout value.